### DISCUSSION TITLE
Using granular access tokens Through npm cli

### DISCUSSION THREAD

**Discussion by tushar-arch** [community/community#49763] [New to GitHub]
Hi folks, 
I want to use the new feature granular access tokens to query the information through npm CLI.

I created a granular access token with read and write permission for the organization. 

But as I was going through the npm documentation here: https://docs.npmjs.com/ 
 there is no command here to use the granular access token in the npm cli to query data . 

I want to know the command used to query data from the npmjs.com using the granular access token 
eg: npm --granular access token [ something here] 

Please help me with this

Thanks in advance

**Comment by queenofcorgis** [comment]
Thanks for your patience @tushar-arch 💟 

You cannot make these through the CLI, using them is the same as any other auth token. Have you seen these [docs](https://docs.npmjs.com/cli/v9/configuring-npm/npmrc#auth-related-configurationthe)? 
 
The token can be configured in an npmrc file and how you do that will depend on where you are running the client.  

One thing to keep in mind is that Granular access tokens cannot do everything that normal access tokens can do. They are also designed primarily for CI/CD use cases. For local cases, it will be best to use a normal publish token.
